modules/network-observability-packet-translation.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * network_observability/observing-network-traffic.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-packet-translation_{context}"]
+= Working with endpoint translation (xlat)
+You can use Network Observability and eBPF to enrich network flows from a Kubernetes service with translated endpoint information, gaining insight into the endpoints serving traffic.
+
+.Procedure
+. In the web console, navigate to *Operators* -> *Installed Operators*.
+. In the *Provided APIs* heading for the *NetObserv Operator*, select *Flow Collector*.
+. Select *cluster*, and then select the *YAML* tab.
+. Configure the `FlowCollector` custom resource for, for `PacketTranslation`, for example:
++
+[id="network-observability-flowcollector-configuring-packet-translation_{context}"]
+.Example `FlowCollector` configuration
+[source, yaml]
+----
+apiVersion: flows.netobserv.io/v1beta2
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+  namespace: netobserv
+  agent:
+    type: eBPF
+    ebpf:
+      features:
+       - PacketTranslation   <1>
+----
+<1> You can start enriching network flows with translated packet information by listing the `PacketTranslation` parameter in the `spec.agent.ebpf.features` specification list.
+ 
+.Example filtering
+When you refresh the *Network Traffic* page you can filter for information about translated packets:
+.. Filter the network flow data based on *Destination kind: Service*.
+.. You can see the *xlat* column, which demarcates where translated information is displayed. For example, the Service IP, *IP*, gets translated to *destination IP* and the destination port, *port*, gets translated to the *destination port*. In a single row, the service traffic and endpoint traffic is consolidated together. 
```

Comment:
For example, the Service IP, *IP*, gets translated to *destination IP*  and the destination port, *port*, gets translated to the *destination port* which is the pod IP and port respectively of the one serving the request

Reply:
Good clarification, added. Thank you!

Reply:
some times the same service flow can be going through dst and src xlation in netobserv this will be showing as two rows one showing dstXlat and another row showing srcXlat when this happens the pkt ips change that is why netobserv treat them as two flows